lang/common/bridge/avro/Header.avsc

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+[
+  /*
+   * Identify the next message in the Java/C# bridge protocol.
+   */
+  {
+    "namespace":"org.apache.reef.bridge.message",
+    "type":"record",
+    "name":"Header",
+    "doc":"Identifies the following message in the Java/C# bridge protocol.",
+    "fields":[
+      {
+        "name":"identifier",
+        "doc":"Identifier of the next message to be read.",
+        "type":"long"
+      },
+      {
+        "name":"className",
+        "doc":"The name of the message class with org.apache.reef.bridge.message. removed.",
```

Comment:
What I should have said is only the class name without the full package name. However, I think to be prevent collisions we should should the fully qualified path name for the class.